test/library/packages/LinearAlgebra/correctness/testElementwise.chpl

```diff
@@ -0,0 +1,58 @@
+use LinearAlgebra;
+use LinearAlgebra.Sparse;
+use BlockDist;
+use LayoutCS;
+use Lists;
+
+config const n = 20;
+
+proc getSparse(d, stride) {
+  var sdom: sparse subdomain(d);
+  var indices: [1..0] 2*d.idxType;
+  for (i,j) in d by stride {
+    indices.push_back((i,j));
```

Comment:
`push_back` on arrays are deprecated on master (but warnings are currently off) could you use the `list` type for `indices` instead? It'll mean
- changing type from array to `list`
- calling `append` instead of `push_back`
- while adding to sdom, call `toArray` on the list.

... or change the sparse domain creation logic to use something other than `array.push_back`

(and I am aware that you may have taken this from my code :( )

Reply:
Here is the list doc: https://chapel-lang.org/docs/master/modules/standard/Lists.html